powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/PowerAuthClient.java

```diff
@@ -146,13 +146,13 @@ public interface PowerAuthClient {
      * @param applicationId             Application ID for which a new CREATED activation should be created.
      * @param maxFailureCount           How many failed attempts should be allowed for this activation.
      * @param timestampActivationExpire Timestamp until when the activation can be committed.
-     * @param otpValidation             Mode that determines in which stage of activation should be additional OTP validated.
+     * @param commitPhase               Specifies when the activation is committed.
      * @param otp                       Additional OTP value.
      * @return {@link InitActivationResponse}
      * @throws PowerAuthClientException In case REST API call fails.
      */
     InitActivationResponse initActivation(String userId, String applicationId, Long maxFailureCount, Date timestampActivationExpire,
-                                          ActivationOtpValidation otpValidation, String otp) throws PowerAuthClientException;
+                                          CommitPhase commitPhase, String otp) throws PowerAuthClientException;
```

Comment:
Ok, but let's review this approach later on. From the binary point of view, this is a trap and we should prefer methods with parameter objects such as `#initActivation(InitActivationRequest)` and deprecate these multi-parameter ones.